.github/workflows/pr-lint.yml

```
@@ -0,0 +1,16 @@
+name: PR Quality Check
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+jobs:
+  check-pr-name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check ticket name conforms to requirements
+        run: 'echo ${{ github.event.pull_request.title }} | grep -i -E -q "^(niad)-[0-9]+: [a-z]+|(dependabot\/)"'
```

Now that it's simplified, we can also note that `dependabot/` also satifies `[a-z]+`, and is thus redundant. (Also, no need to capture `niad`.)
```suggestion
        run: 'echo ${{ github.event.pull_request.title }} | grep -i -E -q "^niad-[0-9]+: [a-z]+"'
```

---

@andrewwa-kainos removed dependabot and capture group from regex